gulpfile.js

```diff
@@ -31,7 +31,9 @@ var analyticsDirectory = '../analytics';
 // Tasks
 gulp.task('default', ['clean', 'quality', 'webpack']);
 
-gulp.task('serve', ['clean', 'quality', 'devpack', 'webpack', 'watch', 'test']);
+gulp.task('serve', ['clean', 'quality', 'devpack', 'webpack', 'watch', 'test', 'e2etest']);
```

Comment:
We shouldn't set `e2etest` to run an `serve` as additional config is required such as installing the Selenium server.